### ISSUE TITLE
load old messages on the chat

### ISSUE THREAD

**Issue by Titogelo** [weSPOT/pim_android#1]
add an option lazy load old messages in the chat.

**Comment by Titogelo** [comment]
initially, when users enter the chat screen, they are at the bottom of the list of messages. If they scroll up all the way to the top they find a button to retrieve old messages on that chat. But here, I encountered a problem. If I do a chatAdapter.notifyDataSetChanged(); to show the old messages retrieved, it scrolls automatically to the top again (confusing the user because it seems like nothing happened but actually all old messages were loaded but skipped due to the scrolling). 

Summing up, the problem is to skip the scroll action while doing chatAdapter.notifyDataSetChanged().
